src/app/core/services/auth/auth.service.ts

```diff
@@ -0,0 +1,38 @@
+import { IUser } from './../../models/user.model';
+import { Injectable } from '@angular/core';
+
+const MOCK_USER: IUser = {
+  firstName: 'Walton',
+  lastName: 'Decker',
+  id: 123
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  public isAuthenticated: boolean;
+  private user: IUser;
+
+  constructor() {
+    this.isAuthenticated = Boolean(this.user);
+  }
+
+  login(): void {
```

Comment:
use, public, private please

Reply:
everywhere possible

Reply:
As far as I know fields and methods are public by default (unless access modifiers are explicitly set), is this a strict rule to use access identificators everywhere? And what is the advantage of this approach, could you explain?